tests/GenerateScriptRegressionTests/GenerateScriptRegressionTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GenerateScriptRegressionTests;
+
+public class PackageData
```

Comment:
There is no need to separate this into a separate class.  I would only do that if the data is utilizing across multiple test classes.

Reply:
Have put them into test class